changelogs/fragments/18-support-dashes-in-username.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - community.routeros.command - added support for a dash (``-``) in username (https://github.com/ansible-collections/community.routeros/pull/18).
+  - community.routeros.facts - added support for a dash (``-``) in username (https://github.com/ansible-collections/community.routeros/pull/18).
```

Sorry I missed that in the first run; we're always using the short form in the changelog so far, so let's keep it for consistency:
```suggestion
  - command - added support for a dash (``-``) in username (https://github.com/ansible-collections/community.routeros/pull/18).
  - facts - added support for a dash (``-``) in username (https://github.com/ansible-collections/community.routeros/pull/18).
```

---

Sure!